README.md

```diff
@@ -1,22 +1,20 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- Kindly fill your README template with the information of your project. 
(Write a descriptive and easy to understand README in business English for every project submitted.)
- Add information for non-technical people on how to download your repo and how to install and run it locally.